backend/adapter_processor_v2/views.py

```
@@ -148,6 +148,9 @@ class AdapterInstanceViewSet(
 ):
     serializer_class = AdapterInstanceSerializer
     pagination_class = OptionalPagination
+    # `pk` tiebreaker keeps paging deterministic when modified_at collides.
+    ordering = ["-modified_at", "pk"]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)